scrappers/newsScrapper.ts

```diff
@@ -0,0 +1,43 @@
+import * as cheerio from "cheerio";
+import { Element as CheerioElement, Cheerio, SelectorType } from "cheerio";
+
+export interface News {
+  id: number
+  title: string,
+  description: string,
+  content?: string,
+  photoUrl: string,
+  date: Date,
+}
+
+export async function scrapNews(): Promise<News[]> {
+  const data = await fetch("https://www.mat.umk.pl/wiadomosci/", { method: "GET" });
+  const $ = cheerio.load(await data.text());
+
+  const news: News[] = [];
+
+  $("section.wiadomosc").each((_, element) => {
+    news.push(createNewsFromSectionTag($(element)));
+  });
+
+  return news;
+}
+
+export async function fillNewsContent(news: News): Promise<News> {
```

Comment:
Nigdzie nie używasz tej funkcji

Reply:
To będzie używane jak już wejdziemy w dany `News`. Wtedy będziemy mogli pobrać jego content.

Reply:
Ok, czyli `description` to ten krótki opis, który jest przed rozwinięciem, nastomiast `content` po. Nie zauważyłem, że te dwa pola to *prawie* to samo. Przy krótszych newsach tak jest. Rozważyłbym może zmianę nazw żeby było widać, że sa do siebie bardzo podobne `content` i `content_summary` albo `content_short`... Zostawiam to Tobie, przemyśl

Reply:
Dobra, przyjrzę się temu. Racja te nazewnictwo może być ciężkie do ogarnięcia.